### ISSUE TITLE
686 Accessibility Issues

### ISSUE THREAD

**Issue by micahchiang** [department-of-veterans-affairs/va.gov-team#9235]
## Description
Some issues have been discovered as we analyze our 686 markup in preparation for our 508 audit. Those should be fixed before we request an audit.

## Consideration
`TAKE A LOOK` For the workflow of adding a spouse, you can enter a start date that is AFTER the termination date ([reference](https://dsvavsp.testrail.io/index.php?/tests/view/6138))

## Tasks
- [x] Fix marriage history header.
    * Axe throws a warning that the header used in marriage history pages doesn't follow single level progression. It skips from h2 to h4. 
    * [This header](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/disability-benefits/686c-674/components/ArrayPageItemSpouseTitle.jsx) should be made an h3
     * And [this one](https://staging.va.gov/view-change-dependents/add-remove-form-686c/686-report-dependent-death/0/additional-information)
- [x] Text inside the element is not included in the accessible name 
Element source
``` HTML
<a aria-label="800. 8 2 7. 1000" href="tel:800 827 1000">800-827-1000</a>
```
Per Jennifer: Note the new aria-label telephone pattern will throw an error in axe-coconut, even when it is implemented properly. The correct pattern should be `aria-label="8 0 0. 8 2 7. 1 0 0 0."`.  There should be a space between each number and periods where there would be a dash.

## Definition of done
- [x] All tasks have been addressed and a PR has been requested

**Comment by jason-gcio** [comment]
@jenstrickland - Jesse and I were chatting about the aria label above and it seems like if the aria label isn't going to help (since its the same as the link text), it could be left out.  We found this: https://www.w3.org/WAI/WCAG21/Techniques/general/G208.html and wanted to give you a heads up on it.

**Comment by jenstrickland** [comment]
The aria-label should be `aria-label="8 0 0. 8 2 7. 1 0 0 0."` — and will not be the same as the link text. This is also a new pattern on VA.gov, there is a new component that was [recently added to the VA React-Formation NPM module](https://dsva.slack.com/archives/CLXEU7Z17/p1591122570138900?thread_ts=1591122570.138900).

**Comment by jason-gcio** [comment]
@jenstrickland - oh ok, great, this helpful.  Thanks again for your help! 🙌
